pkg/services/teams.go

```diff
@@ -155,7 +161,15 @@ func (s teamsService) Send(notification Notification, dest Destination) error {
 	if !ok {
 		return fmt.Errorf("no teams webhook configured for recipient %s", dest.Recipient)
 	}
-	transport := httputil.NewTransport(webhookUrl, false)
+	var idleConnTimeout time.Duration
+	if s.opts.IdleConnTimeout != "" {
+		var err error
+		idleConnTimeout, err = time.ParseDuration(s.opts.IdleConnTimeout)
```

Comment:
```suggestion
		idleConnTimeout, err := time.ParseDuration(s.opts.IdleConnTimeout)
```

Reply:
currently I am facing this warning everywhere as this is inked inside if condition `declared and not used: idleConnTimeout`